external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/KafkaSpout.java

```diff
@@ -139,8 +139,8 @@ public void onPartitionsRevoked(Collection<TopicPartition> partitions) {
 
         @Override
         public void onPartitionsAssigned(Collection<TopicPartition> partitions) {
-            LOG.info("Partitions reassignment. [consumer-group={}, consumer={}, topic-partitions={}]",
-                    kafkaSpoutConfig.getConsumerGroupId(), kafkaConsumer, partitions);
+            LOG.info("Partitions reassignment. [task-ID={}, consumer-group={}, consumer={}, topic-partitions={}]",
```

Comment:
Thanks Srishty.  Regarding `taskId()` being a function name as the justification for `taskID` being the variable name:  in at least one case the name of the function is bad and should be changed.  i.e., KafkaUtils.java's `taskId()` should be `taskPrefix()` or something else.  It's *not* the "Task ID".